### PR TITLE
fortune-kind: init at 0.1.3

### PR DESCRIPTION
**Pull request by cafkafk**
## Description of changes
Homepage: https://github.com/cafkafk/fortune-kind
Changelogs: https://github.com/cafkafk/fortune-kind/blob/main/CHANGELOG.md

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by cafkafk** [comment]
Result of `nixpkgs-review pr 260020` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>fortune-kind</li>
  </ul>
</details>

**Review by samueltardieu — APPROVED** [review]

File: pkgs/by-name/fo/fortune-kind/package.nix [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      Historically, contributions to fortune-mod have had a less-than ideal
+      quality control process, and as such, many of the fortunes that a user may
+      receive from the program read more like cryptic inside jokes, or at the
+      very worst, locker-room banter. One of the major goals of fortune-kind is
+      defining and applying a somewhat more rigorous moderation and editing
+      process to the fortune adoption workflow.
+    '';
+    homepage = "https://github.com/cafkafk/fortune-kind";
+    changelog = "https://github.com/cafkafk/fortune-kind/releases/tag/v${version}";
+    license = licenses.gpl3;
```
Comment (eclairevoyant) [inline pkgs/by-name/fo/fortune-kind/package.nix:65]: gpl3 identifier is deprecated, either use gpl3Plus or gpl3Only

File: pkgs/by-name/fo/fortune-kind/package.nix [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  src = fetchFromGitHub {
+    owner = "cafkafk";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-q4r1Qdyh2L1vTi+CKln+a9lKGgyRAhvg1aJRd0CIEJs=";
+  };
+
+  cargoHash = "sha256-ArVOGWaKbge8nxxGpAT8TttFZYJpuFF5u/tHYC9Qkeo=";
+
+  nativeBuildInputs = [ makeWrapper installShellFiles ];
```
Comment (eclairevoyant) [inline pkgs/by-name/fo/fortune-kind/package.nix:25]:
```suggestion
  nativeBuildInputs = [ makeBinaryWrapper installShellFiles ];
```
slightly preferred if possible, as mentioned in https://nixos.org/manual/nixpkgs/stable/#fun-makeWrapper

File: pkgs/by-name/fo/fortune-kind/package.nix [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+, fortuneAlias ? true
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "fortune-kind";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "cafkafk";
+    repo = pname;
```
Comment (eclairevoyant) [inline pkgs/by-name/fo/fortune-kind/package.nix:18]:
```suggestion
    repo = "fortune-kind";
```
they're semantically different and should be treated as such

File: pkgs/by-name/fo/fortune-kind/package.nix [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      --zsh  man/_fortune-kind
+    mkdir -p $out
+    cp -r $src/fortunes $out/fortunes;
+  '';
+
+  postInstall = ''
+    wrapProgram $out/bin/fortune-kind \
+      --prefix FORTUNE_DIR : "$out/fortunes"
+  ''+ lib.optionalString fortuneAlias ''
+    ln -s fortune $out/bin/fortune
```
Comment (eclairevoyant) [inline pkgs/by-name/fo/fortune-kind/package.nix:50]:
```suggestion
    ln -s fortune-kind $out/bin/fortune
```
I assume you meant this?

**Review by eclairevoyant — CHANGES_REQUESTED** [review]

File: pkgs/by-name/fo/fortune-kind/package.nix (hunk 2 above)
Comment (cafkafk) [reply pkgs/by-name/fo/fortune-kind/package.nix:25]: OHHH didn't know this, that's cool!

**Comment by cafkafk** [comment]
Result of `nixpkgs-review pr 260020` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>fortune-kind</li>
  </ul>
</details>

**Review by eclairevoyant — APPROVED** [review]

**Pull request merged**